pkg/metadataapi/server/server.go

```diff
@@ -9,13 +9,15 @@ import (
 	"github.com/puppetlabs/horsehead/v2/instrumentation/alerts/trackers"
 	"github.com/puppetlabs/relay-core/pkg/metadataapi/server/api"
 	"github.com/puppetlabs/relay-core/pkg/metadataapi/server/middleware"
+	"github.com/puppetlabs/relay-core/pkg/workflow/validation"
 )
 
 type Server struct {
-	auth             middleware.Authenticator
-	errorSensitivity errawr.ErrorSensitivity
-	capturer         trackers.Capturer
-	trustedProxyHops int
+	auth               middleware.Authenticator
+	errorSensitivity   errawr.ErrorSensitivity
+	capturer           trackers.Capturer
+	trustedProxyHops   int
+	specSchemaRegistry validation.SchemaRegistry
```

Comment:
Does it make sense to rename these fields (+ option functions, etc.) since the package is no longer called `spec`? I think `schemaRegistry` by itself seems sufficient.